packages/bahmni-medication-app/src/components/PrescriptionDialog.tsx

```diff
@@ -0,0 +1,145 @@
+import {
+  Button,
+  Column,
+  DatePicker,
+  DatePickerInput,
+  Dropdown,
+  Grid,
+  NumberInput,
+  Row,
+  Search,
+} from '@bahmni/design-system';
+import React from 'react';
+import { Drug } from '../types';
+
+type PrescriptionDialogProps = {
```

Comment:
Can we use interface instead of type? Refer this article which states the difference between type and interface https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases

Reply:
Going with type makes the code readable and understandable. Also, we hope it is possible to achieve whatever interface can do.